js/packages/openinference-instrumentation-llama-index/src/instrumentation.ts

```diff
@@ -89,6 +91,13 @@ export class LlamaIndexInstrumentation extends InstrumentationBase<
           return patchQueryEmbeddingMethod(original, this.tracer);
         });
       }
+
+      if (isLLMPrototype(prototype)) {
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        this._wrap(prototype, "chat", (original): any => {
```

Comment:
is the return type any here just because the variations on the types of "chat" tha can come from the different classes?

Reply:
AdditionalChatOptions and AdditionalMessageOptions. Should I specify the types?
https://ts.llamaindex.ai/api/interfaces/LLMChatParamsNonStreaming

Reply:
I'm assuming you're referring to the "any" return type. The return type would be the function of `(this: unknown, params: LLMChatParamsNonStreaming<object, object>) => Promise<ChatResponse<object>>`